integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingTest.java

```diff
@@ -292,7 +294,7 @@ public void shouldPublishAndConsumeMessageWithTraceAndSpanHeaders() {
 
         // and
         Topic topic = operations.buildTopic(randomTopic("traceSendAndReceiveGroup", "topic2").build());
-        operations.createSubscription(topic, "subscription", HTTP_ENDPOINT_URL);
+        operations.createSubscription(topic, "subscription", remoteService.getUrl().toString());
```

Comment:
Minor: consider overloading `operations.createSubscription()` in a way that the third argument will be URL class. Thanks to this you will not need to call `.toString()` in many test while creating subscription